wouso/games/teamquest/views.py

```diff
@@ -17,6 +17,89 @@
 from forms import *
 
 
+class TeamHubView(ListView):
```

Comment:
I think this should be a DetailView as you are displaying information about one group only